README.md

```diff
@@ -11,13 +11,15 @@ includes [APIs](https://docs.datacommons.org/api/) and visual tools to easily
 explore and analyze data across different datasets without data cleaning or
 joining.
 
-## Using the tool
+## Using the import tool
 
-- Make sure Java 11 is installed (you can install it from [here](https://www.oracle.com/java/technologies/javase-downloads.html#javasejdk))
+- Make sure Java 11+ (but not Java 16) is installed 
```

Comment:
This "not Java 16" might not be a constraint for just running the tool.  Perhaps you can confirm with your version?

Reply:
Confirmed that Java 16 can run the tool & updated the README